### PR TITLE
[LAPI-480] Refactor tokenInfo API

### PR DESCRIPTION
**Pull request by mt-max**
- remove `tokenInfo` unnecessary options and standardize response

### CODE REVIEW

**Review by mt-kenny — APPROVED** [review]
LGTM. 
Also, we might need to inform Integration Team about the changes before it gets merged.

**Comment by mt-kenny** [comment]
@mt-max Should it be better to bump to v3.0.0 if we follow semantic versioning? 
Or is it because no clients are using v2.0.0 yet so we can say this change as a patch?

**Review by mt-skorin — APPROVED** [review]

File: src/api/token-info.ts [hunk 1]
```diff
@@ -45,16 +27,25 @@ export default async function tokenInfo(
...
+      aud: result.resource_server && [result.resource_server + '.getmoneytree.com'],
+      sub: result.uid,
+      scope: result.scopes.join(' '),
+      client_id: result.aud && result.aud.uid,
+      app: result.aud &&
+        result.aud.name && {
+          name: result.aud.name,
+          is_mt: result.is_mt_client,
+        },
+      guest: result.count &&
```
Comment (mt-kenny) [inline src/api/token-info.ts:42]:
```suggestion
      guest: result.country &&
```

**Comment by mt-max** [comment]
Updated

**Review by mt-kenny — APPROVED** [review]

**Pull request merged**